core/src/rpc/clementine.proto

```diff
@@ -270,7 +268,7 @@ message WithdrawErrorResponse {
 
 message WithdrawResult {
   oneof result {
```

Comment:
why do we need this type? we could've reused gRPC Status as the error variant.

I think it should be an empty response from Withdraw, and we can use the Status variants to indicate errors. We have support for wrapping errors with `Status` variants now, if you wrap an error with `.wrap_err(Status::invalid_argument("..."))` and return it, the `From` trait will return the proper error when converting from `BridgeError` back to `Status`.